README.md

```diff
@@ -46,7 +46,7 @@ To create the latest development build for your platform from this source reposi
 * [JDK 17 64-bit][jdk17]
 * [Gradle 7.3+][gradle]
 * make, gcc, and g++ (Linux/macOS-only)
-* [Microsoft Visual Studio][vs] (Windows-only)
+* [Microsoft Visual Studio][vs] or [Microsoft C++ Build Tools][vcbuildtools] (Windows-only)
 
```

Comment:
I agree with your other comment.  We should have sub-bullets that spell out the required components:
* MSVC
* Windows SDK (for your desired platform)
* C++ ATL

or something worded better.